library/js/prescription-autocomplete/prescription-autocomplete.js

```diff
@@ -0,0 +1,129 @@
+/*
+ *  @package   OpenEMR
+ *  @link      http://www.open-emr.org
+ *  @author    Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @copyright Copyright (c )2019. Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ *
+ *
+ */
+
+
+
+$(document).ready(function () {
+    $("#sendScript").toggle();
+    $('#pharmacy-autocomplete').autocomplete({
+
+        source: function (request, response) {
+            // Fetch data
+            let autocompleteUrl = $("#pharmacyselect").data('autocomplete-url');
+            $.ajax({
+                url: autocompleteUrl+'?term='+request.term,
+                type: 'GET',
+                dataType: "json",
+                data: {
+                    search: request.term
+                },
```

Comment:
This is very odd. your url has a get parameter in it (term), but you then set another get parameter via data (search) that are both sending the same thing. Which get parameter is needed? (and ideal to place all get parameters in the data array)

Reply:
I can't say. This is where I am just copying someone and it works the way it is.  I was trying to be sophisticated when I built this. I was following an instructor building another project while I built this. That is why the autocomplete URL is being passed in through the controller and not hardcoded into the js.  https://makitweb.com/jquery-ui-autocomplete-with-php-and-ajax/ iw where I copied the code from.